packages/mask/src/components/InjectedComponents/AutoPasteFailedDialog.tsx

```diff
@@ -1,7 +1,7 @@
 import { useState } from 'react'
 import { useCopyToClipboard } from 'react-use'
 import { useMaskSharedTrans } from '../../utils/index.js'
-import formatDateTime from 'date-fns/format'
+import { format as formatDateTime } from 'date-fns'
```

Comment:
Does auto-completion keep generating code as the original style?

Reply:
no, but I'd like to leave it as-is to keep the change minimal in this PR.